src/redux/books/books.js

```diff
@@ -1,64 +1,60 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
 const ADD_NEW_BOOK = 'bookstore/books/ADD_NEW_BOOK';
 const DELETE_BOOK = 'bookstore/books/DELETE_BOOK';
-const BookArr = [
-  {
-    id: 1,
-    title: 'The Tempest',
-    author: 'William Shakespear',
-    chapter: 4,
-    genre: 'Adventure Fiction',
-  },
-  {
-    id: 2,
-    title: 'The Gods Are Not To Blame',
-    author: 'Ola Rotimi',
-    chapter: 7,
-    genre: 'novel',
-  },
-  {
-    id: 3,
-    title: 'Even One Child',
-    author: 'Moses Akpanbio',
-    chaper: 5,
-    genre: 'novel',
-  },
-  {
-    id: 4,
-    title: 'A Tale of Thoughts',
-    author: 'Uchecukwu Onwu',
-    chapter: 10,
-    genre: 'poetry',
-  },
-  {
-    id: 5,
-    title: 'Dawn of a New Day',
-    author: 'Benedicta Naomi Mbanuzue',
-    chapter: 8,
-    genre: 'poetry',
+const FETCH_BOOKS = 'bookstore/books/FETCH_BOOK';
+
+const BookArr = [];
+export const addedbooks = createAsyncThunk(
+  ADD_NEW_BOOK,
+  async (item) => {
+    await axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/lJao5He3QugK5nNbGykj/books', {
+      item_id: item.id,
+      title: item.title,
+      author: item.author,
+      category: item.category,
+    });
+    return {
+      book: [
+        item.id,
+        [{
+          author: item.author,
+          title: item.title,
+          category: item.category,
+        }],
+      ],
+    };
   },
-];
+);
+export const booksDeletedFromApi = createAsyncThunk(DELETE_BOOK, async (id) => {
+  await axios.delete(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/BZgwqZ2DNziuT17roN8k/books/${id}`);
+  return { id };
+});
+
+// Reducer
+
 const bookReducer = (state = BookArr, action) => {
   switch (action.type) {
-    case ADD_NEW_BOOK:
-      return [
-        ...state, action.book,
-      ];
-    case DELETE_BOOK:
-      return state.filter((book) => book.id !== action.id);
+    case `${FETCH_BOOKS}/fulfilled`:
+      return action.payload.books;
```

Comment:
- [ ] When I load the app the books from the API are not displayed as expected. Kindly correct that.

  I added two books | I reloaded the page
  :---:|:---:
  ![litt](https://user-images.githubusercontent.com/81468675/191528019-bd32ff77-71ea-4452-b6b3-dc14e62614d5.jpg) | ![litt-2](https://user-images.githubusercontent.com/81468675/191528015-4f8d96b0-ec44-4c7b-b899-dffe4007fa25.jpg)